locales/en-US/app.l20n

```diff
@@ -104,8 +103,9 @@
 <contribute "Contribute">
 <nowActive "Now Active">
 <userCount title:"Active Users of this Experiment">
-
 <tourOnboardingTitle "{{title}} enabled!">
 <tourStartButton "Take the Tour">
 <tourCancelButton "Skip">
 <tourDoneButton "Done">
+<userCountContainer "There are <span data-l10n-id="userCount" class="bold" data-hook="install-count"></span> people trying <span data-hook="title"></span> right now!">
```

Comment:
I don't know l20n, but would something like this work?

```
"There are {{userCount}} people trying {{title}} right now!"
```

Adding too much markup in .l20n files seems risky for translators.